pyiceberg/typedef.py

```diff
@@ -199,3 +200,6 @@ def __repr__(self) -> str:
     def record_fields(self) -> List[str]:
         """Return values of all the fields of the Record class except those specified in skip_fields."""
         return [self.__getattribute__(v) if hasattr(self, v) else None for v in self._position_to_field_name]
+
+
+VersionNumber = Literal[1, 2]
```

Comment:
What do you think of:
```suggestion
TableVersion = Literal[1, 2]
```

Reply:
Seems more user-friendly and relatable to the Iceberg world!